content/courses/intro-to-ada/chapters/subprograms.rst

```diff
@@ -64,6 +64,32 @@ This is the implementation of the function above:
        return I + Incr;
     end Increment_By;
 
+.. admonition:: In the GNAT toolchain
+
+   The Ada standard doesn't mandate in which file the specification or the
+   implementation of a subprogram must be stored. In other words, the standard
+   doesn't require a specific file structure or specific file name extensions.
+   For example, we could save both the specification and the implementation of
+   the :ada:`Increment` function above in a file called :file:`increment.ada`.
```

Comment:
Maybe go further, to make the point, and use "increment.txt" rather than an extension that at least looks like a source file name?
You can have the entire source code in one file if you want. Dumb, but legal.
For that matter, from the language's point of view, you could use some other representation instead of a file.